packages/client/lib/__tests__/CredentialRequestClient.spec.ts

```diff
@@ -136,6 +186,28 @@ describe('Credential Request Client ', () => {
       Error(URL_NOT_VALID),
     );
   });
+
+  it('should fail with invalid url', async () => {
+    const credReqClient = CredentialRequestClientBuilder.fromCredentialOfferRequest({ request: INITIATION_TEST })
+      .withCredentialEndpoint('httpsf://oidc4vci.demo.spruceid.com/credential')
```

Comment:
is `httpsf` correct or is the f just a typo?

Reply:
On purpose given the name of the test, but indeed could be more clear